persist/fs/seek.go

```diff
@@ -198,7 +218,16 @@ func (s *seeker) Open(namespace ts.ID, shard uint32, blockStart time.Time) error
 	}
 
 	s.dataFd = dataFd
-	return nil
+
+	s.bloomFilter, err = readBloomFilter(
+		bloomFilterFd,
+		s.bloomFilterFdWithDigest,
+		s.expectedBloomFilterDigest,
+		uint(s.bloomFilterInfo.NumElementsM),
+		uint(s.bloomFilterInfo.NumHashesK),
+	)
```

Comment:
Might want to move this above the keepunreadbuf stanza and if `err != nil` then perform `s.Close(); return err` like the other bailout cases?

Reply:
yeah good call